.binstar.yml

```diff
@@ -2,21 +2,17 @@ package: nb_anacondacloud
 
 platform:
   - linux-64
-  - linux-32
-  - osx-64
-  - win-64
-  - win-32
+  # - linux-32
+  # - osx-64
+  # - win-64
+  # - win-32
 
 engine:
   - python=2.7
   - python=3.4
   - python=3.5
 
-install:
-  - conda build conda.recipe -c anaconda-notebook
-  - conda install nb_anacondacloud --use-local -c anaconda-notebook
-
-test:
-  - python -m unittest discover
+script:
+  - conda build conda.recipe -c anaconda-notebook -c javascript -c mutirri -c jakirkham -c bokeh
```

Comment:
OK, post merge review :wink:

any package needed should be migrated/copy to the new org containing the extensions...